### PR TITLE
Switch to unplugin-icons for icons

### LINKED ISSUE #1587: Need help: Move icons to antfu/unplugin-icons
I recently used https://github.com/antfu/unplugin-icons for a different project. It's a great project which could replaces the icons in dozzle. 

See webpack integration.

### PR DESCRIPTION
**Pull request by Coteh**
Fixes #1587 

Screen Shot 2021-11-06 at 9 59 28 PM

Screen Shot 2021-11-06 at 9 12 37 PM

image

Screen Shot 2021-11-06 at 10 13 08 PM

I ended up pulling icons from 3 different sets as some sets had sleeker looking icons than others. Listed below:

[Material Design Light](https://icon-sets.iconify.design/mdi-light/)
- Chevron Left (Hide Sidebar)
- Chevron Right (Show Sidebar)
- Chevron Double Down (Page Down)
- Magnify (Search)
- Cog (Settings)

[Octicons](https://icon-sets.iconify.design/octicon/)
- Trash 24 (Clear Logs)
- Download 24 (Download Logs)
- Container 24 (Container Listing)

[CoreUI Free](https://icon-sets.iconify.design/cil/)
- Columns (Pin as Column)

Let me know your thoughts.

### CODE REVIEW

File: assets/App.vue [hunk 1]
```diff
@@ -168,6 +174,9 @@ export default {
   position: fixed;
   left: 10px;
   bottom: 10px;
+  & svg {
```
Comment (amir20) [inline assets/App.vue:177]: Do you need this if use class `.icon` from bulma? I think it does it for you.

File: package.json [hunk 2]
```diff
@@ -14,19 +24,12 @@
...
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/amir20/dozzle/issues"
-  },
-  "homepage": "https://github.com/amir20/dozzle#readme",
   "dependencies": {
     "@babel/core": "^7.15.8",
     "@babel/plugin-transform-runtime": "^7.15.8",
+    "@iconify-json/cil": "^1.0.1",
```
Comment (amir20) [inline package.json:30]: I think you can remove these and configure `autoInstall: true` instead for icons. They will be automatically installed.

**Review by amir20 — CHANGES_REQUESTED** [review]
Very nice. Some simple fixes and ready to merge. Just a heads up I will need to merge this back to Vue3 branch.

**Comment by amir20** [comment]
Can you rebase also? I updated GH Actions to correctly test against all incoming PRs.

File: package.json (hunk 2 above)
Comment (Coteh) [reply package.json]: You're right. Done. For some reason it doesn't automatically add octicon set to the lockfile, but the octicon files are there.

**Comment by amir20** [comment]
🎉 and we have tests working for PRs again.

**Comment by amir20** [comment]
For the failed js, you can just stub the icon.

File: assets/App.vue (hunk 1 above)
Comment (Coteh) [reply assets/App.vue]: You're right looks like that will do it. Note that it's going to be a bit smaller now.

image

**Comment by Coteh** [comment]
> For the failed js, you can just stub the icon.

Sorry, how should I do this? Is there a specific area where I can specify that this specific dependency should be stubbed?

**Comment by amir20** [comment]
Let me see if I can figure it out because it involves mocking and stubbing things in jest which you might not be familiar with.

**Comment by amir20** [comment]
This is what I ended up doing if you are curious. https://github.com/amir20/dozzle/commit/4f7cbb7cdfe32e0de387091df40395a8cde10440

**Comment by Coteh** [comment]
Oh, looks simple enough. I've used Jest before but not with a Vue project, nor have I mocked a module in that way before. Good to know, thanks for the help!

**Comment by amir20** [comment]
I'll deploy master to my projects and push a build if all good. 

I was also able to rebase into vue3 branch.

**Comment by Coteh** [comment]
Yeah it looks like tests pass and every icon is mocked, so all good. 👍

**Pull request merged**